public/app/features/visualization/data-hover/DataHoverView.tsx

```diff
@@ -63,7 +62,8 @@ export function getDisplayValuesAndLinks(
   const linkLookup = new Set<string>();
 
   for (const field of orderedVisibleFields) {
-    if (mode === TooltipDisplayMode.Single && columnIndex != null && !field.hovered) {
+    const isFieldHovered = columnIndex === fields.indexOf(field);
```

Comment:
ah looking at this we can probably replaced indexOf check with the `hovered` variable - though maybe that should be renamed to `hoveredFieldIndex` or something for better readability?